ext/easylogging-8.91/easylogging++.h

```diff
@@ -2249,6 +2255,16 @@ class Logger {
         return configured_;
     }
 
+    // no checks performed
+    // target must exist longer, than logger OR be removed
```

Comment:
I do not like that it shall not be owned by logger.
Nobody else needs it.
So, but default logger shall own it and remove it.
Otherwise, we ask people to write more code than needed.